ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/Config.scala

```diff
@@ -618,7 +618,7 @@ object Config {
           .text("Legacy flag with no effect")
           .action((_, config) => config)
 
-        opt[Unit]("buffered-ledger-api-streams-unsafe")
+        opt[Unit]("buffered-ledger-api-streams")
```

Comment:
This looks like a breaking change. At the very least you want to mention that in the changelog.

Reply:
this flag is hidden